packages/config/types/router.d.ts

```diff
@@ -6,9 +6,14 @@
  */
 
 import { RouterOptions, RouteConfig } from 'vue-router'
+import Vue, { ComponentOptions, AsyncComponent } from 'vue'
```

Comment:
You can remove these 3 following added lines (see how in other requested change)